### ISSUE TITLE
Subscribe form interactive states are not being handled

### ISSUE THREAD

**Issue by joeday** [joeday/joedaymusic#8]
There is no feedback, success or otherwise, with the subscribe form. 

Buttondown API here: https://docs.buttondown.com/api-subscribers-create

**Comment by buckrivard** [comment]
Do you want to change over to AJAX for form submission? I think the form submit will redirect to a success page on their side, but you're right that errors aren't handled

**Comment by buckrivard** [comment]
<img width="847" alt="image" src="https://github.com/user-attachments/assets/25c7344e-1652-44b8-9fe1-8b45e713d6ec">
FYI

**Comment by joeday** [comment]
Someone did sign up last week. Successfully. But... there are a few ways that could have happened outside of my website, so it's hard to say with any certainty that the form worked.

**Comment by joeday** [comment]
I wonder if they have test email addresses we can use for this purpose? I'll look into that.

**Comment by joeday** [comment]
For reference, this is how I'd like to handle success:
![image](https://github.com/user-attachments/assets/3558ea6b-34a9-4c79-aa21-f875f33fcdb3)

**Comment by buckrivard** [comment]
Yeah, this will need to be AJAX then. Will take a bit of work but nothing too crazy
